scalafix-core/src/main/scala/scalafix/rewrite/NoExtendsApp.scala

```diff
@@ -14,18 +14,24 @@ case class NoExtendsApp(mirror: Mirror) extends SemanticRewrite(mirror) {
   override def rewrite(ctx: RewriteCtx): Patch = {
     def wrapBodyInMain(template: Template) =
       ctx
-        .templateBodyTokens(template)
-        .map { body =>
+        .templateBodyTokens(template) match {
+        case None =>
+          ctx.reporter.warn(
+            "Missing or empty body for object that extends scala.App. See http://dotty.epfl.ch/docs/reference/dropped/delayed-init.html for possible workarounds.",
+            template.pos
+          )
+          Patch.empty
```

Comment:
You might want to attach a message to a non-empty patch to explain why a particular diff happened. This is not so useful right now since rewrites run in batch mode but it will make more sense if you want to integrate scalafix into IDEs, and create a yellow inspection message saying something like  "Hey you should not use auto-tupling, it's bad!"